EMIP/1-1000/EMIP00171.xml

```diff
@@ -65,6 +69,18 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                        <layout columns="2"/>
                      </layoutDesc>
                   </objectDesc>
+                  <decoDesc>
+                    <decoNote type="miniature" xml:id="d01"><locus target="#2"/><desc>The Holy Trinity <ref type = "authFile" corresp ="AT1030Trinity"/>, with each member holding an orb of authority. The symbols of the four creatures are in the four corners </desc></decoNote>
+                    <decoNote type="miniature" xml:id="d02"><locus target="#4"/><desc>The Madonna and Child, <ref type="authFile" corresp="AT1031VirginAndChild"> attended by two angels <term key="angel"/></ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d03"><locus target="#6"/><desc><persName ref = "PRS4585Georgeo"/> and the <term key = "dragon"/>, <ref type="authFile" corresp="AT1035HolyMan"></ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d04"><locus target="#8"/><desc>The Crucifixion <ref type="authFile" corresp="AT1005Crucifixion"></ref>, John stands on the right with hand raised to head in mourning.
+                    Mary and two other women stand to the left looking down and disconsolate. Mary has arms stretched out and down in dismay </desc></decoNote>
+                    <decoNote type="miniature" xml:id="d05"><locus target="#10"/><desc>Jesus Christ <ref type = "authFile" corresp ="AT1084JesusChrist"/>. The Judgment of the Dead. Jesus stands in center holding a
```

Comment:
all cases where you use a term, and then an empty `<ref/>` element with the corresponding entity, will result in a double name in the running text. please instead of 
```xml
Jesus Christ <ref type = "authFile" corresp ="AT1084JesusChrist"/>
```
use instead 

 ```xml
 <ref type = "authFile" corresp ="AT1084JesusChrist">Jesus Christ</ref>
```

or, if the standard name for the entity may be used, you can simply leave the ref.

 ```xml
 <ref type = "authFile" corresp ="AT1084JesusChrist"/>
```


please check this in all files

https://betamasaheft.eu/Guidelines/?id=decorationDescription